helix-term/src/commands.rs

```diff
@@ -3352,10 +3383,14 @@ fn push_jump(editor: &mut Editor) {
 }
 
 fn goto_line(cx: &mut Context) {
-    if let Some(count) = cx.count {
-        push_jump(cx.editor);
+    goto_line_internal(&mut cx.editor, cx.count)
+}
 
-        let (view, doc) = current!(cx.editor);
+fn goto_line_internal(editor: &mut Editor, count: Option<NonZeroUsize>) {
```

Comment:
There's a convention of suffixing `_impl` to internal implementation like `goto_line_impl` in the codebase btw.

Reply:
Thanks for letting me know. I changed it.